doc/make.py

```diff
@@ -228,15 +278,34 @@ def main():
                            type=str,
                            default=os.path.join(DOC_PATH, '..'),
                            help='path')
+    argparser.add_argument('--docstring',
+                           metavar='FILENAME',
```

Comment:
How much does it complicate things to do `--nargs='*'`? I'm thinking you might have a shared doc and want to do `--docstring Series.fillna DataFrame.fillna`

Reply:
Ah, not, that wouldn't be to hard I think. Good idea